### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by Sushant-Chavan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,7 @@
 Repository for learning.
 
 Created by "Amit-VC"
+
+
+Hello World
+-By Amit
```
Comment (Amit-VC) [inline README.md]: Author should be Sushant

Comment (Amit-VC) [reply README.md]: Thanks for the change.

Comment (Sushant-Chavan) [reply README.md]: Fixed in the new commit

Comment (Amit-VC) [reply README.md]: Thanks for the Changes.

**Pull request merged**